### DISCUSSION TITLE
Terralith and large biomes

### DISCUSSION THREAD

**Discussion by federicoacker** [EnigmaticaModpacks/Enigmatica8#76] [Q&A] [answered]
Hi, I was just wondering if there's a particular way to get terralith to generate large biomes server side or if that's even a possibility in the first place, to have large biomes with terralith generation

**Comment by theboo** [comment, marked as answer]
That's probably a question for terralith discord as we are not currently using terralith.. you can find their discord invite on their curseforge page

**Reply by federicoacker** [reply]
yeah sorry I'm an idiot, I was looking for terralith github and for some reason this popped up in google and I didn't double check the name of the github xD
